ocaml/driver/maindriver.ml

```diff
@@ -113,9 +113,11 @@ let main argv ppf =
     n
   | exception Continue
   | () ->
-    Compmisc.with_ppf_dump ~stdout:() ~file_prefix:"profile"
-      (fun ppf -> Profile.print ppf !Clflags.profile_columns
-        ~timings_precision:!Clflags.timings_precision);
+    (* Prevents outputting when using make install to dump CSVs for whole compiler *)
```

Comment:
Maybe add something like:
"See scripts/profile-compiler-build.sh for a use case".

Otherwise, I am not sure it is
obvious to the reader why.